src/index.scss

```diff
@@ -452,3 +452,28 @@ $popup-delay: .3s;
     color: $color-primary !important;
     text-decoration: none !important;
 }
+
+.glimpse-logs {
+    svg {
+        height: 12px !important;
+        width: 12px !important;
+        fill: rgba(white, 0.5) !important;
```

Comment:
I thought we want to keep HUD as widely supported as possible vs (2 last versions for CLient). Good to know.